### PR TITLE
Build all images on pull requests

### PR DESCRIPTION
**Pull request by zoetrope**
And, fix minimal tag for noble.

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        with:
+          go-version: ${{ env.go-version }}
+      - name: Build images
+        run: |
+          TAG_MINIMAL=$(cat TAG_MINIMAL)
+          TAG=$(cat TAG)
+          
+          echo
+          echo "building ubuntu-minimal:$TAG_MINIMAL..."
+          docker build \
```
Comment (takahiro-yamada) [inline .github/workflows/build.yaml:29]: [Q]
It's only for test so it needn't to run `buildx`, right?

File: 24.04/ubuntu-debug/Dockerfile [hunk 2]
```diff
@@ -13,7 +13,6 @@ FROM ghcr.io/cybozu/ubuntu:${TAG}
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
         apache2-utils \
-        awscli \
```
Comment (takahiro-yamada) [inline 24.04/ubuntu-debug/Dockerfile:16]: `awscli` is used and it should be included in containers.
How about installing it following the official procedures?
Additionally, you might need to run `apt-get install unzip`.

```sh
&& curl "https://awscli.amazonaws.com/awscli-exe-linux-$(uname -m).zip" -o "awscliv2.zip" \
&& unzip awscliv2.zip \
&& ./aws/install \
&& rm -f awscliv2.zip
```

**Review by takahiro-yamada — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yaml (hunk 1 above)
Comment (zoetrope) [reply .github/workflows/build.yaml:29]: Actually, when using buildx, I don't know how to use the image built in the runner to build another image.
If you know, please

File: 24.04/ubuntu-debug/Dockerfile (hunk 2 above)
Comment (zoetrope) [reply 24.04/ubuntu-debug/Dockerfile:16]: I took it.

File: .github/workflows/build.yaml (hunk 1 above)
Comment (takahiro-yamada) [reply .github/workflows/build.yaml:29]: Sorry, me too. I think it's ok because it's just a build test and `release.yaml` runs `buildx`.

**Pull request merged**